src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -273,6 +273,7 @@ bool mpuGyroReadSPI(gyroDev_t *gyro)
         // We need some offset from the gyro interrupts to ensure sampling after the interrupt
         gyro->gyroDmaMaxDuration = 5;
         if (gyro->detectedEXTI > GYRO_EXTI_DETECT_THRESHOLD) {
```

Comment:
(don't worry about this one. I did not see the code duplication at first)

This `#ifdef` style is used in BF, but it is quite confusing. In this case, it can be constructed as sequence of tests with decreasing priority, with `break;` when done (and `do {} while (0);` block can be used for that purpose outside of switch( 
); 

But does not work that well in nested-if case :-(

```
 if (gyro->detectedEXTI > GYRO_EXTI_DETECT_THRESHOLD) {
#if USE_DMA
    if (spiUseDMA(&gyro->dev)) {
        ...
        break;
    }
#endif
    gyro->gyroModeSPI = GYRO_EXTI_INT;
    break;
}
gyro->gyroModeSPI = GYRO_EXTI_NO_INT;
break